osu.Game/Screens/Play/ReplayPlayer.cs

```diff
@@ -14,6 +15,12 @@ public ReplayPlayer(Replay replay)
             Replay = replay;
         }
 
+        [BackgroundDependencyLoader]
+        private void load()
+        {
+            HudOverlay.KeyCounter.Visible.Value = true;
```

Comment:
is this necessary? isn't it already handled above?

Reply:
what's the one above for then?

Reply:
The one in Player itself you mean? That's for Autoplay, since that doesn't go through a ReplayPlayer.

Reply:
![image](https://user-images.githubusercontent.com/191335/41285385-e9c33db4-6e76-11e8-8c95-9d55c7137db7.png)

can't we just use the fact that this is already a `Bindable`?

Reply:
Indeed that can be done, nice catch.